web/routes/general.py

```diff
@@ -41,3 +41,15 @@ def monitor(device_id):
                            pins_config=server_interface.get_device_pin_config(device_id),
                            socket_url=WEB_SERVICE_CONFIG["host"],
                            socket_port=WEB_SERVICE_CONFIG["socket_port"])
+
+
+@web_service.route("/index", methods=["GET"])
+@login_required
+@roles_accepted("admin")
```

Comment:
@roles_accepted decorator accepts both single and multiple values.
Move both roles into a single decorator call (i.e. @roles_accepted("user", "admin"))